pkg/client/unversioned/client.go

```diff
@@ -114,6 +115,12 @@ func (c *Client) ComponentStatuses() ComponentStatusInterface {
 	return newComponentStatuses(c)
 }
 
+// WatchOptions is passed as a parameter to per-resource Watch method.
+type WatchOptions struct {
```

Comment:
Sure - we can selectively do 1 as we go though (replace WatchOptions with ListOptions which should have timeout, etc)